django_celery_beat/schedulers.py

```diff
@@ -119,6 +118,14 @@ def is_due(self):
                 )
                 return schedules.schedstate(False, delay)
 
+        # EXPIRED TASK: Disable task when expired
+        if self.model.expires is not None:
```

Comment:
@auvipy does this work for expire_seconds?
Is this check not intended for `self.model.expires_`?